web/client/epics/context.js

```diff
@@ -137,14 +138,16 @@ const createSessionFlow = (mapId, contextName, action$, getState) => {
  */
 export const loadContextAndMap = (action$, { getState = () => { } } = {}) =>
     action$.ofType(LOAD_CONTEXT).switchMap(({ mapId, contextName }) => {
+        const isPermalink = isPermalinkSelector(getState());
+        const resourceCategory = isPermalink ? 'PERMALINK' : 'CONTEXT';
```

Comment:
Looking at the code I'm wandering what happens after I open the permalink if I navigate to the context. Permalink id stays there and the session may be calculated wrongly. Please make a double check on this.

by the way seeing `session` in epics of context shows that the functionalities are not properly decoupled, maybe this needs to be improved in the future.

Reply:
Upon load finishes, we clear permalink hash. So it loads fine when another context is loaded.

But I have another question, now that permalink is redirected to load `context/hash`, upon refreshing the page, the call is fired as context for permalink hash which will result in `Resource not found` is this okay? Or should we instead save the permalink hash in a localstorage and compare hash to determine the category?